test/e2e/conformance_test.go

```diff
@@ -166,6 +157,14 @@ func TestClusterConformance(t *testing.T) {
 				t.Fatalf("failed to download kubeconfig failed ('kubeone kubeconfig'): %v", err)
 			}
 
+			// Run Terraform again for GCE to add nodes to the load balancer
```

Comment:
well, for E2E it will be perfectly find to run cluster only on 1 node behind the LB.

Reply:
I think we should test as we would do in the real environment. This adds some more lines, but it's a proper way.